src/graphqlHelpers.ts

```diff
@@ -97,7 +97,7 @@ const scalarMap: Record<string, OutType> = {
   Boolean: { kind: "scalar", type: "boolean" },
   GitHubGitObjectID: { kind: "scalar", type: "string" },
   GitHubURI: { kind: "scalar", type: "string" },
-  // JSON: "JSON",
+  JSON: { kind: "scalar", type: "Record<string, unknown>" },
```

Comment:
This isn't accurate, unfortunately. We need a `JSONObject` custom scalar, since `JSON` could be _any_ JSON object.